back/boxtribute_server/graph_ql/definitions/public_api.graphql

```diff
@@ -84,6 +85,35 @@ type TopProductsDimensions {
   category: [DimensionInfo]
 }
 
+type MovedBoxesData implements DataCube {
```

Comment:
I would rename it. MovedBoxes is a bit misleading. Since we only take a look at how Boxes are moved out of the warehouse and we neglect internal moves.

Reply:
outgoingBoxes